spiped/main.c

```diff
@@ -274,50 +284,92 @@ main(int argc, char * argv[])
 			warnp_syslog(1);
 	}
 
+	/* Resolve bind address. */
+	if (opt_b) {
+		char * a = strchr(opt_b, ':');
```

Comment:
Sorry for the delay, and thanks for this addition!  I've extracted the "add a :0 if necessary" into a new PR #358, since we'll need to use this code for both spiped/main.c and spipe/main.c.